tests/Pinta.Effects.Tests/Mocks/MockPalette.cs

```diff
@@ -0,0 +1,18 @@
+using Cairo;
+using Pinta.Core;
+
+namespace Pinta.Effects.Tests;
+
+internal sealed class MockPalette : IPaletteService
+{
+	public Color PrimaryColor { get; set; } = new (0, 0, 0); // Black
+	public Color SecondaryColor { get; set; } = new (255, 255, 255); // White
+
+	public void SetColor (bool setPrimary, Color color, bool addToRecent = true)
```

Comment:
Because the Utilities class is creating a singleton palette, this could be a future source of bugs - if tests try to set the color then this might cause the changes to affect other test runs. (And tests are executed in parallel)
I'd maybe lean towards not caching the mock service manager for now since it's currently very cheap to construct

Reply:
Understood. I'm now creating an instance every time